src/js/keyboardInput.js

```diff
@@ -198,6 +207,7 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
     gpii.firstDiscovery.keyboardInput.registerKeypressListener = function (that, input, keymap) {
         input.keypress(function (e) {
             e.preventDefault();
+            e.stopPropagation();
```

Comment:
After a discussion in the IRC channel we decided to go with the explicit approach, so this code is fine - we should, for consistency, be on the lookout for any uses of "return false" in our codebase of which I'm sure there are many

https://botbot.me/freenode/fluid-work/2015-05-15/?msg=39275771&page=1